apps/validate/validate.py

```diff
@@ -65,6 +67,12 @@ def _validate_type_embed(self, field, value):
         """Allow type media in schema."""
         pass
 
+    def _validate_type_date(self, field, value):
+        try:
+            datetime.strptime(value or '', '%d/%m/%Y')
```

Comment:
we use `%Y-%m-%d` usually, why this format here?

Reply:
This was the default format on frontend, I changed on frontend to send always in  %Y-%m-%d format.